Cargo.toml

```diff
@@ -4,4 +4,5 @@ members = [
   "core",
   "tools/hyper_hello",
   "deno_typescript",
+  "cli/tests/test_native_plugin"
```

Comment:
This directory `cli/tests/` is special - I would try to move test_native_plugin to the root for now and see if that fixes thing.